src/main/webapp/app/view/project/admin/users/project-user-invite.component.ts

```diff
@@ -1,4 +1,4 @@
-import { Component } from '@angular/core';
+import { ChangeDetectorRef, Component } from '@angular/core';
```

Comment:
I cannot see any usages of this `ChangeDetectorRef`?

Reply:
woops